ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -158,6 +139,29 @@ class ClientQueryViewController: QueryFileListTableViewController, UIDropInterac
 		leaveMultipleSelection()
 	}
 
+	private func updateTitle() {
+		let lastPathComponent = (query.queryPath as NSString?)!.lastPathComponent
+
+		if lastPathComponent.isRootPath, let shortName = core?.bookmark.shortName {
+			self.navigationItem.title = shortName
+		} else {
+			let titleButton = UIButton()
```

Comment:
@hosy do we need this during selection (button to present breadcrumb UI)? Otherwise setting title textAttributes would be probably just fine

Reply:
No, this was set after the selection did end. Otherwise the title bar gets not updated.

Reply:
@hosy ok